test/functional/cursor.test.js

```diff
@@ -4117,58 +4119,64 @@ describe('Cursor', function() {
     }
   });
 
-  it('Correctly decorate the collection cursor count command with skip, limit, hint, readConcern', {
-    // Add a tag that our runner can trigger on
-    // in this case we are setting that node needs to be higher than 0.10.X to run
-    metadata: {
-      requires: { topology: ['single', 'replicaset', 'sharded', 'ssl', 'heap', 'wiredtiger'] }
-    },
+  // FIXME(NODE-3074): should this be deleted?
+  it.skip(
```

Comment:
Maybe NODE-3369 should be made more general. I looked into python and it looks like they only use the aggregation method of counting if a certain wire version is met, but I guess the tests don't/can't cover that.